### PR TITLE
changelog: Assemble changes for 21.0 release

### PR DESCRIPTION
**Pull request by tjanez**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,1197 @@ The format is inspired by [Keep a Changelog].
...
+- go/epochtime: Add `ConsensusParameters` method to epochtime backend
+  ([#3565](https://github.com/oasisprotocol/oasis-core/issues/3565))
+
+- go/consensus: Add `EpochtimeConsensusParameters` to the consensus backend
+  ([#3565](https://github.com/oasisprotocol/oasis-core/issues/3565))
+
+  This is temporary since we currently do not provide an epochtime
+  gRPC service. The `ConsensusParameters` method on the new beacon backend
+  can replace this method.
+
```
Comment (kostko) [inline CHANGELOG.md:492]: We can remove this.

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,1197 @@ The format is inspired by [Keep a Changelog].
...
+- go/worker/storage: Add `StorageRPC` role and change access policy
+  ([#3696](https://github.com/oasisprotocol/oasis-core/issues/3696))
+
+  Getting checkpoints and diffs is now allowed for any connecting node,
+  which eliminates a race condition and initialization difficulty in the
+  storage committee node startup.
+
+  State access is made gated but optionally public as before, depending on
+  command line parameters.
```
Comment (kostko) [inline CHANGELOG.md:530]: We should maybe publish what parameter can be used to enable this?

File: CHANGELOG.md [hunk 3]
```diff
@@ -12,6 +12,1197 @@ The format is inspired by [Keep a Changelog].
...
+  ([#3308](https://github.com/oasisprotocol/oasis-core/issues/3308))
+
+  Previously, an `ErrInvalidArgument` was returned in this case, which
+  is not very descriptive, so a separate error was created:
+  `ErrUnderMinDelegationAmount`.
+
+- go/worker/p2p: Include chain ID and remove patch version from topic IDs
+  ([#3311](https://github.com/oasisprotocol/oasis-core/issues/3311))
+
+  This is a breaking `RuntimeCommitteeProtocol` change.
```
Comment (ptrus) [inline CHANGELOG.md:77]: I would remove this line (since the category is already breaking)

File: CHANGELOG.md (hunk 2 above)
Comment (tjanez) [reply CHANGELOG.md:530]: Yes, can you provide the details to put here?

Comment (kostko) [reply CHANGELOG.md:530]: Yes maybe like this:
```suggestion
  State access is made gated but optionally public as before, depending on
  the value of the `worker.storage.public_rpc.enabled` configuration option.
```

**Review by ptrus — APPROVED** [review]

**Review by kostko — APPROVED** [review]

**Pull request merged**